### DISCUSSION TITLE
Any way to have hand-crafted bookmark file, per audio/video file ?

### DISCUSSION THREAD

**Discussion by bdutta** [smplayer-dev/smplayer#107] [Q&A]
Is there a way by which one could craft a bookmark file where there is a time-stamp and heading, that can be changed/loaded per audio/video file ? Something like:
00:03:00 "Main introduction"
00:04:15 "Deep dive"
00:06:35 "Summary"
00:07:10 "Conclusion"

The key thing is that such a bookmark file (similar to lyrics file or subtitles file) is file based that can be loaded into SMPlayer along with the audio/video content file ?

**Comment by bernd-wechner** [comment]
I'd be content to work out how SMplayer currently remember bookmarks (which it clearly does yet I can't find any clue as to how or where on MS Windows).
